.gitpod.Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM gitpod/workspace-full

```

Comment:
We've been having an issue with this gitpod file in the other standalones to date and it had to be drastically simplified.

I would suggest using one of those Dockerfiles as reference for now.

Going to add this into my tasking list as something we need to investigate of what is going on. Probably good to add to the containerization feature currently in draft.